### ISSUE TITLE
Crash on middle mouse button

### ISSUE THREAD

**Issue by FishbaitHarry** [atom/atom#6285]
Atom version: 0.189.0
Operating system: Linux Mint 17 Cinnamon 64-bit
Installed from the .deb package.

After using the editor for a while (~10 minutes) the moment I press the middle mouse button (to paste some text into editor) the whole editor crashes. The issue does not appear if I just open a new window and start pasting content, only after some changes have been done to file. I ran the editor with '-f' option and looked into developer tools but no error of any kind appears in logs.

![a not-really-informative screenshot](https://cloud.githubusercontent.com/assets/1315472/7027754/9bc240a6-dd51-11e4-951a-b336d9ed07d6.png)

**Comment by izuzak** [comment]
This looks very similar to #6221 which makes me think it's a duplicate of that issue. That issue should be fixed in the next release of Atom (probably out later today), but please let us know if you can still reproduce this in 0.190.0 once it's released! Thanks for the report.
